### PR TITLE
Translate words from Russian to other Scribe languages

### PR DESCRIPTION
**Pull request by shashank-iitbhu**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

- Added `translate_words.py` to `src/scribe_data/extract_transform/languages/Russian/translation`
- This PR can be used as a reference for other translation related issues.
- This PR can be merged after merging #88 
 
### Related issue

- #77

### CODE REVIEW

File: src/scribe_data/extract_transform/languages/Russian/translations/translate_words.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+words_to_translate_path = os.path.join(translate_script_dir, 'words_to_translate.json')
+
+with open(words_to_translate_path, 'r', encoding='utf-8') as file:
+    json_data = json.load(file)
+
+word_list = {}
+for item in json_data:
+    word_list.append(item["word"])
+
+src_lang="Russian"
```
Comment (shashank-iitbhu) [inline src/scribe_data/extract_transform/languages/Russian/translations/translate_words.py:22]: Modify the `src_lang` variable to specify the source language for the translations. 
Keep the rest of the `translate_words.py` script unchanged.

**Comment by shashank-iitbhu** [comment]
This PR can be tested by running `python3 src/scribe_data/extract_transform/languages/Russian/translations/translate_words.py` from the project root.

This will also test #88.
cc @andrewtavis @wkyoshida

File: src/scribe_data/extract_transform/languages/Russian/translations/translate_words.py (hunk 1 above)
Comment (andrewtavis) [reply src/scribe_data/extract_transform/languages/Russian/translations/translate_words.py:21]: 🔥🔥🔥 Nice, @shashank-iitbhu :) I'll take a bit more of a look at this, and maybe we can move this to an all caps var at the top. I can handle this myself though 😊

**Comment by andrewtavis** [comment]
Note that the Linux tests don't work if we don't have `language-data` in the dependencies, which is a bit weird as it should be installed with `langcodes`... The Mac build in general is pretty broke at this point, as the step to install the PyICU dependencies is what errored now. We can focus on this a bit more after the structure and dependencies are a bit more set :)

**Review by andrewtavis — APPROVED** [review]
Thanks so much for the contributions here, @shashank-iitbhu! Great stuff 😊 I already switched over the English translation process to match the Russian one, and will direct the others to send along their own versions and run it once for the examples. Lots of changes coming along to change the structure, and I'll also finalize the initial docs in #57 as we can't automodule/autodoc a lot of our files as they're scripts :)

**Pull request merged**